src/mono/wasm/runtime/package-lock.json

```diff
@@ -518,7 +518,7 @@
     },
     "concat-map": {
       "version": "0.0.1",
-      "resolved": "https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public-npm/npm/registry/concat-map/-/concat-map-0.0.1.tgz",
+      "resolved": "https://registry.npmjs.org/concat-map/-/concat-map-0.0.1.tgz",
```

Comment:
is this compliant for dependency policies in our repo?

Reply:
this file should not be in the diff, I will revert it